src/pages/Account/Tabs/ModulesTab.tsx

```diff
@@ -78,5 +162,31 @@ export default function ModulesTab({address}: ModulesTabProps) {
     return <Error address={address} error={error} />;
   }
 
-  return <ModulesContent data={data} />;
+  if (!module) {
+    return <EmptyTabContent />;
+  }
+
+  return (
+    <Box>
+      <Typography>ABI</Typography>
+      <JsonViewCard data={module.abi} />
+    </Box>
+  );
+}
+
+type ModulesTabProps = {
+  address: string;
+  accountData: Types.AccountData | undefined;
+};
+
+export default function ModulesTab({address}: ModulesTabProps) {
+  const inDev = useGetInDevMode();
+  if (inDev) {
+    return <ModulesContent2 address={address} />;
+  }
+  return <ModulesContent address={address} />;
+}
+
+function transformCode(source: string): string {
```

Comment:
let's move it to the helper file? i would imagine this could be used in multiple places in the future